currency_exchange_rates/currency_exchange_rates.yml

```diff
@@ -0,0 +1,363 @@
+---
+apiVersion: influxdata.com/v2alpha1
+kind: Label
+metadata:
+    name: Exchange Rates
+spec:
+    color: '#7A65F2'
+---
+apiVersion: influxdata.com/v2alpha1
+kind: Label
+metadata:
+    name: outputs.influxdb_v2
```

Comment:
this name doesn't feel very descriptive 🤔

Reply:
adding labels for the plugins being used is useful for filtering. I added more for the other plugins being used.

Reply:
🤔 how does one filter by label? Is that pkger specific?

Reply:
you can do it in the UI.

Reply:
oh wow, had no idea this was possible 👏

Reply:
I copied that from another template so I assumed this is how I should do it.

Thank you Russ for adding the remaining labels.